src/components/Profile/components/MembershipsList/index.js

```diff
@@ -82,6 +82,18 @@ const MembershipsList = ({ user, myProf, createSnackbar }) => {
     return <GordonLoader />;
   }
 
+  const transcriptButton = !PersonType?.includes('fac')
+    ? myProf && (
+        <Grid container justifyContent="center">
+          <Link className="gc360_link" to="/transcript">
+            <Button variant="contained" className={styles.memberships_card_content_button}>
+              Experience Transcript
+            </Button>
+          </Link>
+        </Grid>
+      )
+    : null;
```

Comment:
Mixing the ternary operator with `&&` is syntactically odd and can produce unexpected results. Include `myProf` in the boolean for the ternary instead:
```suggestion
  const transcriptButton = myProf && !PersonType?.includes('fac')
    ?  (
        <Grid container justifyContent="center">
          <Link className="gc360_link" to="/transcript">
            <Button variant="contained" className={styles.memberships_card_content_button}>
              Experience Transcript
            </Button>
          </Link>
        </Grid>
      )
    : null;
```

Reply:
done

Reply:
Did this change get reverted somehow?